src/main/daml/Daml/Finance/Holding/Base.daml

```diff
@@ -11,13 +11,13 @@ import Daml.Finance.Interface.Util.Lockable qualified as Lockable (I, Lock(..),
 import Daml.Finance.Util.Disclosure (addObserversImpl, removeObserversImpl, setObserversImpl)
 import Daml.Finance.Util.Lockable (acquireImpl, isValidLock, releaseImpl)
 
--- | Type synonym for `NonFungibleNonTransferable`.
-type T = NonFungibleNonTransferable
+-- | Type synonym for `Base`.
+type T = Base
 
 -- | Implementation of a non-transferable holding.
--- `NonFungibleNonTransferable` implements the interface `Base.I` (which requires `Lockable.I` and
+-- `Base` implements the interface `Base.I` (which requires `Lockable.I` and
```

Comment:
`Base` here might be unclear, I think we just call it the `Holding` interface otherwise

Reply:
Agree, but let's rename in a follow up PR as discussed. 

If we use `Holding` we would get a module name called `module Daml.Finance.Interface.Holding.Holding` which might be fine or we find a different name for the base holding.